src/containers/NFT/NFTHeader/Details.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { CopyToClipboard } from '../../shared/components/CopyToClipboard'
+import './styles.scss'
+import { formatPrice } from '../../shared/utils'
+
+interface Props {
+  data: {
+    NFTId: string
+    ledgerIndex: number | undefined
```

Comment:
https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#optional-properties

these should all be defined using optional properties:

```typescript
data: {
  NFTId: string
  ledgerIndex?: number
  ...
}
```